atlassian_jwt_auth/contrib/flask_app/decorators.py

```diff
@@ -1,7 +1,14 @@
+import warnings
+
 from atlassian_jwt_auth.frameworks.flask.decorators import with_asap
 
 
 def requires_asap(f, issuers=None, subject_should_match_issuer=None):
+    warnings.warn(
```

Comment:
perhaps` deprecated as of version 4.0.0;` ?

Reply:
Technically, it's deprecated immediately as of the moment it's committed. :+1: 
If anything, we should be telling people when it will be discontinued completely.